packages/shared-ux/chrome/navigation/src/ui/components/navigation_section_ui.tsx

```diff
@@ -107,6 +107,15 @@ export const NavigationSectionUI: FC<Props> = ({
 
   const groupHasLink = Boolean(navNode.deepLink) || Boolean(navNode.href);
 
+  useEffect(() => {
+    // We only want to set the collapsed state during the "mounting" phase (500ms).
```

Comment:
I think related https://github.com/elastic/kibana/pull/159906#discussion_r1235233847

Reply:
> Does this mean that initially "isActive" is incorrect? Why so?

I am not sure it would be possible or would require quite a complex logic. 

The state order is:
* `isActive: false`
* item register themselves in their parents
* the root `Navigation` component register the tree in the ChromeService
* the ChromeService updates the `activeNodes` based on the navigation tree + the current location
* the active nodes are returned to the `Navigation`
* each node checks if it is active and updates the `isActive` state

That's why all node initially mount as `isActive: false` and only after they are mounted the `activeNodes` array is constructed.

> is it possible to track if the section was manually opened and in that case not auto-collapse it or something like this?

It is not only when it was manually opened, also if it was opened automatically on page load because one of the children is active.

You can disable the `if` statement around `setIsCollapsed(!isActive);` and see for yourself why it is needed.

Reply:
Basically we don't want to close automatically a group that has been opened (manually or automatically).